lib/chef/knife/core/subcommand_loader.rb

```diff
@@ -23,7 +23,7 @@ class Knife
     class SubcommandLoader
 
       MATCHES_CHEF_GEM = %r{/chef-[\d]+\.[\d]+\.[\d]+}.freeze
-      MATCHES_THIS_CHEF_GEM = %r{/chef-#{Chef::VERSION}/}.freeze
+      MATCHES_THIS_CHEF_GEM = %r{/chef-#{Chef::VERSION}(-x86-mingw32)?/}.freeze
```

Comment:
Sure. I looked into https://rubygems.org/gems/chef/versions and found only "-x86-mingw32" gems, but that might change in the future.

I think it would be safe to relax the pattern a bit so we will catch 64bit builds as well once we have them.